app/models/user.rb

```diff
@@ -5,6 +5,9 @@ class User < ActiveRecord::Base
   require 'net/http'
   require 'digest/md5'
 
+  enum dashboard_sort:   [:sort_by_date, :sort_by_group]
+  enum dashboard_filter: [:show_all, :show_unread]
```

Comment:
Enums are the sexy new thing!

I think I might replace them with a string and a model validation.